### PR TITLE
Update example run command in tutorial

### PR DESCRIPTION
**Pull request by wenleix**
It is not recommended to run `mlflow ui` from the root directory of
MLFlow. As a result, the tutorial is updated to use `examples` directory
as the working directory. The two example commands are however not
updated accordingly.

## What changes are proposed in this pull request?

Update the example commands in tutorial document to reflect the working directory is `examples`.
(The working directory is updated in https://github.com/mlflow/mlflow/pull/289)

### Is this a user-facing change?

- [X] No. You can skip the rest of this section.
- [ ] Yes. Give a description of this change to be included in the release notes for MLflow users.

(Details in 1-2 sentences. You can just refer to another PR with a description if this PR is part of a larger change.)

### What component(s), interfaces, languages, and integrations does this PR affect?
Components 
- [ ] `area/artifacts`: Artifact stores and artifact logging
- [ ] `area/build`: Build and test infrastructure for MLflow
- [ ] `area/docs`: MLflow documentation pages
- [x] `area/examples`: Example code
- [ ] `area/model-registry`: Model Registry service, APIs, and the fluent client calls for Model Registry
- [ ] `area/models`: MLmodel format, model serialization/deserialization, flavors
- [ ] `area/projects`: MLproject format, project running backends
- [ ] `area/scoring`: Local serving, model deployment tools, spark UDFs
- [ ] `area/server-infra`: MLflow server, JavaScript dev server
- [ ] `area/tracking`: Tracking Service, tracking client APIs, autologging

Interface 
- [ ] `area/uiux`: Front-end, user experience, JavaScript, plotting
- [ ] `area/docker`: Docker use across MLflow's components, such as MLflow Projects and MLflow Models
- [ ] `area/sqlalchemy`: Use of SQLAlchemy in the Tracking Service or Model Registry
- [ ] `area/windows`: Windows support

Language 
- [ ] `language/r`: R APIs and clients
- [ ] `language/java`: Java APIs and clients
- [ ] `language/new`: Proposals for new client languages

Integrations
- [ ] `integrations/azure`: Azure and Azure ML integrations
- [ ] `integrations/sagemaker`: SageMaker integrations
- [ ] `integrations/databricks`: Databricks integrations

<a name="release-note-category"></a>
### How should the PR be classified in the release notes? Choose one:

- [ ] `rn/breaking-change` - The PR will be mentioned in the "Breaking Changes" section
- [x] `rn/none` - No description will be included. The PR will be mentioned only by the PR number in the "Small Bugfixes and Documentation Updates" section
- [ ] `rn/feature` - A new user-facing feature worth mentioning in the release notes
- [ ] `rn/bug-fix` - A user-facing bug fix worth mentioning in the release notes
- [ ] `rn/documentation` - A user-facing documentation change worth mentioning in the release notes

### CODE REVIEW

**Comment by harupy** [comment]
@wenleix Thanks for the PR! The proposed changes make sense. Can we add `cd examples` somewhere in the tutorial to clarify where we should run the `sklearn_elasticnet_wine` example?

File: docs/source/tutorials-and-examples/tutorial.rst [hunk 1]
```diff
@@ -78,13 +78,13 @@ First, train a linear regression model that takes two hyperparameters: ``alpha``
 
     .. code-block:: py
 
-        python examples/sklearn_elasticnet_wine/train.py
+        python sklearn_elasticnet_wine/train.py
```
Comment (harupy) [inline docs/source/tutorials-and-examples/tutorial.rst:81]: Can we insert a comment like this?

```suggestion
        # Make sure the current working directory is 'examples'
        python sklearn_elasticnet_wine/train.py
```

Comment (wenleix) [reply docs/source/tutorials-and-examples/tutorial.rst:81]: @harupy : Makes sense. In fact that's what exactly happened to me 😂

Comment (wenleix) [reply docs/source/tutorials-and-examples/tutorial.rst:82]: @harupy : Added. Thanks for the suggestions.

**Comment by harupy** [comment]
@wenleix Could you fix the DCO check failure follwoing the instruction here: https://github.com/mlflow/mlflow/pull/4116/checks?check_run_id=1932858747?

File: docs/source/tutorials-and-examples/tutorial.rst (hunk 1 above)
Comment (harupy) [reply docs/source/tutorials-and-examples/tutorial.rst:82]: Thanks for the quick fix :)

**Comment by wenleix** [comment]
@harupy  Thanks, comments addressed. Test failure is irrelevant.

**Review by harupy — APPROVED** [review]
Looks good to me :)

**Comment by harupy** [comment]
@wenleix I've just merged the master to fix the broken check. I'll merge after all the CI checks turn green. Thanks for the contribution!

**Pull request merged**